src/realm/sync/noinst/client_history_impl.cpp

```diff
@@ -1231,6 +1232,19 @@ void ClientHistoryImpl::trim_sync_history()
     do_trim_sync_history(n); // Throws
 }
 
+bool ClientHistoryImpl::is_clean(version_type version) const
```

Comment:
What does "clean" mean? Could it be more descriptive perhaps?

Reply:
I don't want to be too specific about what "clean" means.

Reply:
Something like `has_pending_local_changes`?

Reply:
Ended up with `no_pending_local_changes`.